source/user-manual/capabilities/active-response/remediation-faq.rst

```diff
@@ -11,7 +11,21 @@ FAQ
 
 Can I use a custom script for active responses?
 -----------------------------------------------
-Yes. You can create your own script and configure a command and active response to refer to it.
+Yes. You can create your own script and configure a command and active response to refer to it. Keep in mind that AR follows a specific arguments syntax when running scripts. The arguments are inserted in this order:
+
+.. code-block:: none
+
+  <SCRIPT-NAME> <ACTION> <USER> <IP> <ALERT-ID> <RULE-ID> <AGENT> <FILENAME>
+
+Some considerations:
+
+* ``<SCRIPT-NAME>`` is the name of the script file that is going to be run.
+* ``<ACTION>`` can be *delete* or *add*.
+* ``<USER>`` is the user name. It can be *-* if not set.
+* ``<IP>`` is the source IP. It can be *-* if not set.
+* ``<ALERT-ID>`` is the alert ID (unique for every alert).
+* ``<RULE-ID>`` is the rule ID.
+* ``<AGENT>`` is the agent ID or hostname.
```

Comment:
Please, add to this list the `filename` explanation. It is the source file of the log that triggered the alert in case of it exists.

Reply:
Done here @chemamartinez ! https://github.com/wazuh/wazuh-documentation/pull/661/commits/c2b47f80aae7f3c9efc669cba1eb8667b02304b2